src/org/exist/plugin/PluginsManagerImpl.java

```diff
@@ -78,21 +83,31 @@
 	private Configuration configuration = null;
 	
 	private Collection collection;
-	
+
 	private Database db;
-	
-	public PluginsManagerImpl(Database db, DBBroker broker) throws ConfigurationException {
-		this.db = db;
-		
+
+	@Override
+	public void prepare(final BrokerPool brokerPool) {
+		this.db = brokerPool;
+
 		//Temporary for testing
 		addPlugin("org.exist.scheduler.SchedulerManager");
 		addPlugin("org.exist.storage.md.MDStorageManager");
 		addPlugin("org.exist.monitoring.MonitoringManager");
 	}
 
 	@Override
+	public void startSystem(final DBBroker systemBroker) throws BrokerPoolServiceException {
+		try {
+			start(systemBroker);
+		} catch(final EXistException e) {
+			throw new BrokerPoolServiceException(e);
+		}
+	}
+
+	@Override
 	public void start(DBBroker broker) throws EXistException {
-        final TransactionManager transaction = db.getTransactionManager();
+        final TransactionManager transaction = broker.getBrokerPool().getTransactionManager();
```

Comment:
@shabanovd That is the name you gave it when you created this code, I didn't change the name. Would you like me to fix it?